recipes/libvdwxc/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "libvdwxc" %}
+{% set version = '0.4.0' %}
+{% set build_number = 0 %}
+
+{% if mpi != 'nompi' %}
+  {% set mpi_prefix = "mpi_" + mpi %}
+{% else %}
+  {% set mpi_prefix = "nompi" %}
+{% endif %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://launchpad.net/{{ name }}/stable/{{ version }}/+download/{{ name }}-{{ version }}.tar.gz
+  sha256: 3524feb5bb2be86b4688f71653502146b181e66f3f75b8bdaf23dd1ae4a56b33
+
+build:
+  skip: True  # [not unix]
+  number: {{ build_number }}
+  string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build_number }}"
+  run_exports:
+    {% if mpi != 'nompi' %}
+    - {{ name }} * {{ mpi_prefix }}_*
+    {% endif %}
+
```

```suggestion
  ignore_run_exports:
    - libgfortran5
 
```
This is how you ignore run dependencies from packages in build or host.